openassessment/management/commands/performance_test_for_submissions.py

```diff
@@ -1,74 +0,0 @@
-"""
-Gives the time taken by
-    find_active_assessments
-    get_submission_for_review
-    get_submission_for_over_grading
-    methods for particular set of workflows.
-"""
-import random
-import datetime
-
-from django.core.management.base import BaseCommand
-
-from openassessment.assessment.models import PeerWorkflow
-
-
-class Command(BaseCommand):
```

Comment:
I yanked out all performance tests as well, as they also heavily blurred the line between "python library" and "suite of scripts and programs to install, some of which may be python". The tests that were here were incredibly old, I think if we need performance testing against ora we're better off rewriting it in https://github.com/edx/edx-load-tests using the modern locust setup described there.